main.tf

```diff
@@ -25,7 +25,7 @@ resource "aws_subnet" "default" {
   cidr_block        = "${cidrsubnet(length(var.base_cidr) > 0 ? var.base_cidr : data.aws_vpc.default.cidr_block, ceil(log(local.subnets_count * 2, 2)), count.index)}"
 
   tags = {
-    "Name"      = "${module.subnets_label.id}${var.delimiter}${element(var.names, count.index)}${var.delimiter}${element(var.availability_zones, count.index)}"
+    "Name"      = "${module.subnets_label.namespace}-${module.subnets_label.stage}-${format("%v-%v", element(var.names, count.index, element(var.availability_zones, count.index))}"
```

Comment:
What about the other `tags`? anywhere we use `subnets_count` should be the same

Reply:
I've changed back `module.subnets_label.namespace` to `module.subnets_label.id`, lets do all cosmetic fixes in next release.